streams/functions/src/main/java/com/hortonworks/streamline/streams/udf/Ltrim.java

```diff
@@ -0,0 +1,19 @@
+package com.hortonworks.streamline.streams.udf;
+
+import com.hortonworks.streamline.streams.rule.UDF;
+
+/**
+ * Trims leading white spaces from the input.
+ * See {@link String#trim()}
+ */
+public class Ltrim implements UDF<String, String> {
+    @Override
+    public String evaluate(String input) {
+        int len = input.length();
+        int st = 0;
+        while ((st < len) && (input.charAt(st) <= ' ')) {
```

Comment:
For `TRIM` I return the value of `java.lang.String.trim` which trims `<= ' '`. So I wanted to be consistent for ltrim and rtrim. This has the advantage that tabs, newline, carriage return etc also gets trimmed.

Maybe we can keep this as the default and add overloaded methods that also accepts the character to trim.